<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>405</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
    <addaction name="actionGenerateSamples"/>
    <addaction name="separator"/>
    <addaction name="actionBaseline"/>
    <addaction name="actionLinearRegression"/>
    <addaction name="actionNearestNeighbor"/>
    <addaction name="actionLocalRegression"/>
    <addaction name="actionClusteredLR"/>
    <addaction name="actionMCMC"/>
    <addaction name="separator"/>
    <addaction name="actionFindLinearity"/>
    <addaction name="actionAngleEffect"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTool"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionGenerateSamples">
   <property name="text">
    <string>Generate Samples</string>
   </property>
  </action>
  <action name="actionLinearRegression">
   <property name="text">
    <string>Linear Regression</string>
   </property>
  </action>
  <action name="actionGenerateSampleFiles">
   <property name="text">
    <string>Generate Sample Files</string>
   </property>
  </action>
  <action name="actionNearestNeighbor">
   <property name="text">
    <string>Nearest Neighbor</string>
   </property>
  </action>
  <action name="actionLocalRegression">
   <property name="text">
    <string>Local Regression</string>
   </property>
  </action>
  <action name="actionBaseline">
   <property name="text">
    <string>Baseline</string>
   </property>
  </action>
  <action name="actionMCMC">
   <property name="text">
    <string>MCMC</string>
   </property>
  </action>
  <action name="actionFindLinearity">
   <property name="text">
    <string>Find Linearity</string>
   </property>
  </action>
  <action name="actionClusteredLR">
   <property name="text">
    <string>Clustered LR</string>
   </property>
  </action>
  <action name="actionAngleEffect">
   <property name="text">
    <string>Angle Effect</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
